whelktool/scripts/2023/01/lxl-3908-romanized-to-byLang/bib880.groovy

```diff
@@ -3,125 +3,160 @@ import whelk.util.DocumentUtil
 
 Util u = new Util(getWhelk())
 
+romanized = getReportWriter("romanized.tsv")
+missingRomanized = getReportWriter("missing-romanized.tsv")
+unsafePath = getReportWriter("unsafe-path.tsv")
+lost = getReportWriter("lost-values.tsv")
+looksLikeLatin = getReportWriter("looks-like-latin.tsv")
+
 def where = """
     collection = 'bib'
     AND data#>>'{@graph,1,${u.HAS_BIB880}}' IS NOT NULL
 """
 
 selectBySqlWhere(where) {
```

Comment:
This block is a bit too long for the implicit `it` to be easily tracked; name it explicitly (and perhaps factor out more of the code body)?